politeiawww/cmsaddresswatcher.go

```diff
@@ -33,6 +33,10 @@ func (p *politeiawww) addWatchAddress(address string) {
 	if err != nil {
 		log.Errorf("addWatchAddress: subscribe '%v': %v",
 			address, err)
+		p.reconnectWS()
+		if p.wsDcrdata != nil {
```

Comment:
Are these `if p.wsDcrdata != nil` checks necessary?  Won't cmswww always have a wsDcrdata context?

If cmswww is suppose to always have a wsDcrdata context then the nil checks aren't necessary because if for some reason wsDcrdata is nil then something is wrong and we would want it to panic.

Reply:
Ah yah I believe that was a remnant from previous versions.  Basically with the recursion there, I wanted to ensure that we weren't repeatedly trying to addWatchAddress without having a wsDcrdata set.  Though now the reconnect logic repeatedly attempts until it connects. Will remove the above check.